docs/user-guide/system-admin/configuration/installation.md

```diff
@@ -95,6 +95,31 @@ This setting will cause Zanata to output CORS headers for REST requests, includi
 `Access-Control-Allow-Origin` (if Origin is in the whitelist above),
 `Access-Control-Allow-Credentials: true` and
 `Access-Control-Allow-Methods: PUT,POST,DELETE,GET,OPTIONS`.
+
+
+## Redirecting HTTP to HTTPS
```

Comment:
It's not really. I built this on top of the other branch, which hasn't been merged yet.